src/core/stdc/config.d

```diff
@@ -112,6 +122,9 @@ version (Windows)
 
     alias long  cpp_longlong;
     alias ulong cpp_ulonglong;
+
+    enum __c_wchar_t : wchar;
+    alias wchar_t = __c_wchar_t;
```

Comment:
I strongly suggest aliasing directly - `alias wchar_t = wchar`. `__c_[u]long[long]` are reserved identifiers; these magic enums (previously: structs) are solely needed to recognize these types (as opposed to a regular `[u]{int,long}`) in the frontend for C++ mangling purposes, and not needed here.

Reply:
They are magic https://github.com/dlang/dmd/commit/4ba6c2a46f02d3662c9e289ef5454fc261ecd8d3

Reply:
Why?! Can't `{w,d}char` be simply mangled appropriately for `extern(C++)`?

Reply:
they need to mangled as `char[16|32]_t`, see https://github.com/dlang/dmd/pull/8342

Reply:
Hence `wchar_t` needs its own.

Reply:
Ah, thx for the link. C++ is just such a mess.

Reply:
True dat.